src/enclave_proc/resource_manager.rs

```diff
@@ -597,6 +614,164 @@ impl EnclaveHandle {
+                    let des_sign: Vec<PcrSignature> = from_slice(&buf[..]).map_err(|e| {
+                        new_nitro_cli_failure!(
+                            &format!("Error deserializing certificate: {:?}", e),
+                            NitroCliErrorEnum::EifParsingError
+                        )
+                    })?;
+
+                    let cert = openssl::x509::X509::from_pem(&des_sign[0].signing_certificate)
+                        .map_err(|e| {
+                            new_nitro_cli_failure!(
+                                &format!("Error while digesting certificate: {:?}", e),
+                                NitroCliErrorEnum::EifParsingError
+                            )
+                        })?;
+                    let cert_der = cert.to_der().unwrap();
+
+                    // Write the hash and save it to measurements only if
+                    // signature section is present
+                    cert_hasher.write_all(&cert_der).unwrap();
+                    let cert_hasher = hex::encode(
+                        cert_hasher
+                            .tpm_extend_finalize_reset()
+                            .expect("Could not get result for cert_hasher"),
+                    );
+                    self.measurements.insert("8".to_string(), cert_hasher);
+                }
+                EifSectionType::EifSectionInvalid => {
+                    new_nitro_cli_failure!(
+                        "Eif contains an invalid section",
+                        NitroCliErrorEnum::EifParsingError
+                    );
+                }
+            }
+        }
+
+        let image_hasher = hex::encode(
+            image_hasher
+                .tpm_extend_finalize_reset()
+                .expect("Could not get result for image_hasher"),
+        );
+        let bootstrap_hasher = hex::encode(
+            bootstrap_hasher
+                .tpm_extend_finalize_reset()
+                .expect("Could not get result for bootstrap_hasher"),
+        );
+        let app_hasher = hex::encode(
+            app_hasher
+                .tpm_extend_finalize_reset()
+                .expect("Could not get result for app_hasher"),
+        );
+        self.measurements.insert("0".to_string(), image_hasher);
+        self.measurements.insert("1".to_string(), bootstrap_hasher);
+        self.measurements.insert("2".to_string(), app_hasher);
+
+        self.print_pcrs_at_run(connection)?;
+        Ok(())
+    }
+
+    fn print_pcrs_at_run(&mut self, connection: Option<&Connection>) -> NitroCliResult<()> {
+        safe_conn_eprintln(
+            connection,
+            &format!(
+                "PCR0: {}\nPCR1: {}\nPCR2: {}",
+                self.measurements.get(&"0".to_string()).unwrap(),
+                self.measurements.get(&"1".to_string()).unwrap(),
+                self.measurements.get(&"2".to_string()).unwrap()
+            )[..],
+        )?;
```

Comment:
These values should be printed as part of the JSON output, similar to `describe-enclaves`.